webserver/src/main/java/io/kestra/webserver/events/PluginHotReloadInitializer.java

```diff
@@ -0,0 +1,29 @@
+package io.kestra.webserver.events;
+
+import io.kestra.core.contexts.KestraApplicationContext;
+import io.kestra.webserver.controllers.PluginController;
+import io.micronaut.cache.CacheManager;
+import io.micronaut.context.ApplicationContext;
+import io.micronaut.context.event.ApplicationEventListener;
+import io.micronaut.discovery.event.ServiceReadyEvent;
+import jakarta.inject.Inject;
+import jakarta.inject.Singleton;
+
+import java.util.Optional;
+
+@Singleton
+public class PluginHotReloadInitializer implements ApplicationEventListener<ServiceReadyEvent> {
```

Comment:
Can't you do this in th AbstractCommand?
Also, is it possible that a CacheManager didn't exist?

Reply:
Nope because AbstractCommand doesn't have the application context initialized yet so I have no injection point